app/Models/Task.php

```diff
@@ -14,4 +15,18 @@ class Task extends Model
         'description',
         'status'
     ];
+
+    public function user(): BelongsTo
+    {
+        return $this->belongsTo(User::class, 'user_id', 'id');
+    }
+    public function toArray()
+    {
+        return[
+            'name'=>$this->getAttribute('name'),
+            'description'=>$this->getAttribute('description'),
+            'status'=>$this->getAttribute('status'),
+            'user'=>$this->getRelationValue('user')
```

Comment:
Call user toArray() here as below, so that it will give only the required details. Now, you can add many tasks on tests.

` 'user'=> $this->getRelationValue('user') !== null ($this->getRelationValue('user'))->toArray() : null`